### PR TITLE
Audit more with fewer threadstorms

### PR DESCRIPTION
**Pull request by VictorLowther**

### CODE REVIEW

File: rails/app/models/user.rb [hunk 1]
```diff
@@ -20,6 +20,9 @@ class User < ActiveRecord::Base
   # :authenticatable, :confirmable, :database_authenticatable, :lockable, :omniauthable, 
   # :recoverable, :registerable, :rememberable, :timeoutable, :token_authenticatable, 
   # :trackable, :validatable
+
+  audited
+
```
Comment (galthaus) [inline rails/app/models/user.rb]: May want to change this one to:
audited except: [:password, :encrypted_password]

I think that syntax is right.

**Comment by galthaus** [comment]
I'm fine with this pending the results around the one comment.

Future work will need:
  * get the UI view of the audit table
  * create a bridge from rails to AMQP (This will reenable the AMQP functionality)
  * Add addition audit actions into the log for login events

**Comment by sjensen** [comment]
look at the audited table in the db

**Comment by sjensen** [comment]
great now we need to decide what information is useful ;)

**Pull request merged**